### PR TITLE
 【Hackathon No.61】min 算子FP16/BF16单测完善

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Others
### Describe

min OP add FP16 and BF16

### CODE REVIEW

**Comment by ZzSean** [comment]
改下PR Title

File: python/paddle/fluid/tests/unittests/test_reduce_op.py [hunk 1]
```diff
@@ -418,6 +418,51 @@ def test_check_output(self):
         self.check_output()
 
 
+@skip_check_grad_ci(
```
Comment (ZzSean) [inline python/paddle/fluid/tests/unittests/test_reduce_op.py:421]: 无需跳过反向

Comment (enkilee) [reply python/paddle/fluid/tests/unittests/test_reduce_op.py:421]: 2023-04-18 21:26:15 ======================================================================
2023-04-18 21:26:15 ERROR: tearDownClass (test_reduce_op.TestMinFP16Op)
2023-04-18 21:26:15 ----------------------------------------------------------------------
2023-04-18 21:26:15 Traceback (most recent call last):
2023-04-18 21:26:15   File "/workspace/Paddle/build/python/paddle/fluid/tests/unittests/eager_op_test.py", line 395, in tearDownClass
2023-04-18 21:26:15     "This test of %s op needs check_grad." % cls.op_type
2023-04-18 21:26:15 AssertionError: This test of reduce_min op needs check_grad.
2023-04-18 21:26:15 
2023-04-18 21:26:15 ----------------------------------------------------------------------

Comment (enkilee) [reply python/paddle/fluid/tests/unittests/test_reduce_op.py:421]: @ZzSean 政哥，看来不加，还是没法判断不执行反向。还是要加的吧？

Comment (enkilee) [reply python/paddle/fluid/tests/unittests/test_reduce_op.py:421]: 我看 NO_FP64_CHECK_GRAD_OP_LIST 有 reduce_min，可是eager_op_test却去判断 NO_FP16_CHECK_GRAD_OP_LIST，那我把reduce_min加进去可行吗？

Comment (ZzSean) [reply python/paddle/fluid/tests/unittests/test_reduce_op.py:421]: 可以加进去

Comment (enkilee) [reply python/paddle/fluid/tests/unittests/test_reduce_op.py:421]: 我加了后，还是报错要反向，只能加这个skip了

**Review by ZzSean — APPROVED** [review]
LGTM

**Pull request merged**